backend/src/v5/middleware/dataConverter/inputs/teamspaces/projects/models/containers.js

```diff
@@ -40,6 +40,21 @@ Containers.canDeleteContainer = async (req, res, next) => {
 	}
 };
 
+Containers.checkModelStatus = async (req, res, next) => {
+	try {
+		const { teamspace, container } = req.params;
+		const { status } = await getModelById(teamspace, container, { _id: 0, status: 1 });
+
+		if (status === 'queued' || status === 'processing') {
+			throw templates.revisionProcessed;
```

Comment:
this status code seems redundant since it'll always get caught to return with invalidArguments?

Reply:
true but we want the message right ? shall I just do throw  'A revision is already being processed.' ?

Reply:
Sure, but I think if you do it through the other middleware and we just fit it into the Yup schema it becomes a non issue